tier3/{{cookiecutter.project_slug}}/README.md

```diff
@@ -4,6 +4,9 @@
 ## About the Project
 **{project statement}**
 
+### Project Vision
```

Comment:
Let's add this section commented out (recommended/optional) for projects outside of tier2?